requirements.txt

```diff
@@ -3,4 +3,4 @@ selenium==3.12.0
 tldextract==2.2.0
 xvfbwrapper==0.2.9
 GitPython==2.1.11
-PyFunceble>=1.8.0
+tranco>=0.3
```

Comment:
We should pin the version of the package to the one we tested with to avoid breakage in the future if the package changes in incompatible ways.